nautobot/docs/plugins/development.md

```diff
@@ -261,7 +263,85 @@ class SiteAnimalCount(PluginTemplateExtension):
         })
 
 
-template_extensions = [SiteAnimalCount]
+class DeviceExtraTabs(PluginTemplateExtension):
+    """Template extension to extra tabs to the object detail tabs."""
+
+    model = 'dcim.site'
```

Comment:
Mismatch between class name and class `model` here.

Reply:
Very well spotted! Thanks!